cmd/mothership/mothership/start/integration_test.go

```diff
@@ -470,16 +470,6 @@ func TestMothership(t *testing.T) {
 			responseModel:    &keb.HTTPClusterResponse{},
 			verifier:         requireClusterStateChangeFct("ready"),
 		},
-		{
```

Comment:
To Reviewer:

The reason why I delete this test case is because this delete request only mark cluster to be deleted, which adds a record in `inventory_cluster_config_statuses` to indicate the runtimeid of this cluster be `delete_pending`, but the record in `inventory_clusters` will not change, and this test we don't have component reconciler to do actual deleting, so eventually the data will not reflect this cluster is deleted.

That's the reason why I add actual cleanup code in the above defer function.